app/src/api/arrangementSvc.ts

```diff
@@ -49,10 +50,15 @@ export const getEvents = (): Promise<WithId<IEventViewModel>[]> =>
     path: `/events`,
   });
 
-export const deleteEvent = (eventId: string, editToken?: string) =>
+export const deleteEvent = (
+  eventId: string,
+  cancellationMessage: string,
+  editToken?: string
+) =>
   del({
     host: getArrangementSvcUrl(),
     path: `/events/${eventId}${queryStringStringify({ editToken })}`,
+    body: replaceNewLinesWithMarkupForEmail(cancellationMessage),
```

Comment:
Hvorfor ønsker vi ikke å gjøre dette backend? Vi gjør det allerede backend for melding ved ny påmelding.

Reply:
Fornuftig, da gjer vi det